decidim-core/lib/decidim/form_builder.rb

```diff
@@ -729,7 +729,7 @@ def abide_error_element(attribute)
       options = { count: 1, default: defaults }
 
       text = I18n.t(defaults.shift, **options)
-      content_tag(:span, text, class: "form-error")
+      content_tag(:span, text, class: "mt-2 hidden peer-invalid:block text-sm text-red")
```

Comment:
@Crashillo we should create CSS components in cases like this. In fact, why not create it as .form-error?